common/src/main/java/com/microsoft/identity/common/internal/providers/oauth2/WebViewAuthorizationFragment.java

```diff
@@ -475,6 +491,10 @@ private HashMap<String, String> getRequestHeaders(final Bundle state) {
         }
     }
 
+    public ActivityResultLauncher<LegacyFido2ApiObject> getFidoLauncher() {
```

Comment:
where is this invoked?

Reply:
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/cef6e97bf8ae3da4badd6b59bcbfed4e63358aba/common/src/main/java/com/microsoft/identity/common/internal/fido/LegacyFido2ApiManager.kt#L89